scripts/jenkins-acceptance-tests.sh

```diff
@@ -37,3 +37,4 @@ pip install -r requirements/test-acceptance.txt
 
 cd test/acceptance
 python tests.py
+SELENIUM_BROWSER=phantomjs python accessibility.py
```

Comment:
Only the accessibility ones would run under phantomjs.  (The line above this runs the others with the default firefox browser.)